evm/its.js

```diff
@@ -79,7 +84,42 @@ const decodeMulticallData = async (encodedData, contractJSON) => {
     });
 };
 
-async function processCommand(_, chain, options) {
+function compare(contractValue, configValue, variableName) {
+    contractValue = isNonEmptyString(contractValue) ? contractValue.toLowerCase() : contractValue;
+    configValue = isNonEmptyString(configValue) ? configValue.toLowerCase() : configValue;
+
+    if (contractValue === configValue) {
+        printInfo(`Confirmed: Value match for '${variableName}'.`);
```

Comment:
```suggestion
        printInfo(`${variableName}`, contractValue);
```
normal print can be considered success